lib/ColorModeButton/index.jsx

```diff
@@ -0,0 +1,29 @@
+import LightModeIcon from '@mui/icons-material/LightModeOutlined'
+import DarkModeIcon from '@mui/icons-material/DarkModeOutlined'
+import AutoModeIcon from '@mui/icons-material/Brightness4'
+import IconButton from '@mui/material/IconButton'
+import { ColorModeContext } from '../GlobalTheming'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+export default function ColorModeButton() {
+  const { t, i18n } = useTranslation()
+  const { toggle, current } = React.useContext(ColorModeContext)
+  const modeIcons = {
+    dark: DarkModeIcon,
+    light: LightModeIcon,
+  }
+  const Icon = modeIcons[current] || AutoModeIcon
+  return (
+    <>
```

Comment:
Seems fragment is not necessary since there is only a child element inside

Reply:
I don't understand the comment, sorry.

Reply:
If so, this has to be done here in order to be used as component inside the JSX. `<Icon />`
I guess you cannot do `<{modeIcons[current] || AutoModeIcon} />` and if you do, it is not as much as clear.

Reply:
got it!!

Reply:
removed